### PR TITLE
[Osdev 1027] Fix displaying of the average lead time in facility details page

### PR DESCRIPTION
**Pull request by VadimKovalenkoSNF**
Fix https://opensupplyhub.atlassian.net/browse/OSDEV-1027

1. Fix appearence of the Average Lead Time in facility page.
2. Update Redux Store - that will give the possibility to test smart components with predefined state easillier.
3. Add unit test for theFacilityDetailsClaimedInfo component.

### CODE REVIEW

File: src/react/src/__tests__/components/FacilityDetailsClaimedInfo.test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import FacilityDetailsClaimedInfo from '../../components/FacilityDetailsClaimedInfo';
+import renderWithProviders from '../../util/testUtils/renderWithProviders';
+
+describe('FacilityDetailsClaimedInfo component', () => {
+    test('test facility details presence', () => {
+      const data = {
```
Comment (VadimKovalenkoSNF) [inline src/react/src/__tests__/components/FacilityDetailsClaimedInfo.test.js:8]: Note, that we use here the `data` object as it was fetched by the request and passed to the component. FacilityDetailsClaimedInfo doesn't interact with Redux state.

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by vladsha-dev — APPROVED** [review]
LGTM

File: src/react/src/__tests__/components/FacilityDetailsClaimedInfo.test.js (hunk 1 above)
Comment (vladsha-dev) [reply src/react/src/__tests__/components/FacilityDetailsClaimedInfo.test.js:8]: Thanks

File: src/react/src/components/FacilityDetailsClaimedInfo.jsx [hunk 2]
```diff
@@ -62,7 +62,7 @@ export default function FacilityDetailsClaimedInfo({ data, formatListItem }) {
                 label="Minimum Order"
             />
             <ClaimInfoSection
-                value={facility.average_lead}
+                value={facility.average_lead_time}
```
Comment (vladsha-dev) [inline src/react/src/components/FacilityDetailsClaimedInfo.jsx:65]: I don't see such a name in serializers.

**Review by vladsha-dev — CHANGES_REQUESTED** [review]

File: src/react/src/components/FacilityDetailsClaimedInfo.jsx (hunk 2 above)
Comment (VadimKovalenkoSNF) [reply src/react/src/components/FacilityDetailsClaimedInfo.jsx:65]: @vladsha-dev check [here](https://github.com/opensupplyhub/open-supply-hub/blob/main/src/django/api/serializers/facility/approved_facility_claim_serializer.py#L35)

Comment (VadimKovalenkoSNF) [reply src/react/src/components/FacilityDetailsClaimedInfo.jsx:65]: Basically, error was on the FE part. Average lead time correctly saves to the database.

**Review by vladsha-dev — APPROVED** [review]
LGTM

File: src/react/src/components/FacilityDetailsClaimedInfo.jsx (hunk 2 above)
Comment (vladsha-dev) [reply src/react/src/components/FacilityDetailsClaimedInfo.jsx:65]: Ok, thanks!

**Pull request merged**